src/scanomf.d

```diff
@@ -187,7 +187,8 @@ void scanOmfObjModule(void delegate(const(char)* name, int pickAny) pAddSymbol,
                 parseName(&p, name.ptr);
                 p += (recTyp == PUBDEF) ? 2 : 4; // skip offset
                 parseIdx(&p); // skip type index
-                pAddSymbol(name.ptr, 0);
+                const len = strlen(name.ptr);
+                pAddSymbol(name[0 .. len], 0);
```

Comment:
>I had it as two-step because it's simpler to read.

Nope, this is wrong. When the reviewer sees a name being introduced, she starts to worry about the use of that name later in the function. In such cases the rvalue is more local so net objectively easier to read.